assets/video_looper.ini

```
@@ -118,6 +118,19 @@ copyloader = false
 password = videopi
 
 
+[playlist]
+# This setting allows for a fixed playlist.
+
+# Path to the playlist.
+# Leave empty to get all files in the directory / USB drive.
+#path = playlist.m3u
```

for better visibility and consistency the default should be:
```suggestion
path =
#path = playlist.m3u
```